### ISSUE TITLE
Feature request: support thrustlever with reverse zone

### ISSUE THREAD

**Issue by merspieler** [legoboyvdlp/A320-family#6]
### General
----

I am submitting a: (select one) 
* [ ] Bug report
* [x] Feature request

### Details
----
#### Requested Behavior  (feature requests)

Meaning, that part of the trustlever range is configured as reverse zone which controls the reverse thrust. The other dentents should be rearranged accordingly.

**Comment by legoboyvdlp** [comment]
Just to confirm, idle should be approximately 25% throttle, so if you throttle down further with the reverse levers up you'll get reverse?

I would need to confirm that would work with throttle hardware, other than that sounds fine in theory.

**Comment by merspieler** [comment]
See as well the few posts in the forum after my [initial request](https://forum.flightgear.org/viewtopic.php?f=4&t=30928&p=334153&hilit=+reverse#p334137)

Josh had an idea how this could work.

**Comment by legoboyvdlp** [comment]
Oh dear - that FADEC code is a horrible mess, not to be touched with a six foot pole... But still, I'll investigate it.

**Comment by legoboyvdlp** [comment]
@Octal450 
![image](https://user-images.githubusercontent.com/10434911/107039725-2a84ff00-67b6-11eb-888d-c1071507ba6c.png)

**Comment by Octal450** [comment]
ok am now looking at this... its possible yes but that'd be really weird to use imo. Doesn't really make sense.
Can you tell me about your setup?

**Comment by merspieler** [comment]
For levers on the joystick or the like, it doesn't make sense, that's true.
But for something like [this](https://shop.thrustmaster.com/en_gb/tca-quadrant-airbus-edition.html) it does make sense.

**Comment by Octal450** [comment]
No, usually such setups they use a separate axis for reverse or a button or something. So are you sure?
Kind Regards,
Josh

**Comment by merspieler** [comment]
Afaik, they have one axis per lever and a "virtual button" that probably triggers with the detent.

[This thread](https://forum.flightgear.org/viewtopic.php?f=24&t=38387&start=45) has some xmls which seem to make it work correctly but there are(or were?) issues with detent alignment....

Unfortunately I haven't got one yet (out of stock everywhere) so I couldn't try my self just yet.

**Comment by merspieler** [comment]
Okay, so I've got another notice from a user who has the TCP quadrant:

The detent buttons don't line up with the actual detents so you've got a thrust change on A/THR DISC

> Nur der Bereich bis zum CL Detent ist bei den TCA Schubhebeln leider etwas klein geraten. Beim 1127G springt er dann bei ner N1 von knapp über 50 zu leicht in den MCT Detent und gibt somit zuviel Schub. Manchmal auch nur auf einer Engine. Da ist noch Luft nach oben, man kann sich aber daran gewöhnen.
en: 
> Only the range up to the CL Detent is unfortunately somewhat small with the TCA thrust levers. With the 1127G, it then jumps too easily into the MCT detent at an N1 of just over 50 and thus gives too much thrust. Sometimes also only on one engine. There is still room for improvement, but you can get used to it.

**Comment by Octal450** [comment]
> The detent buttons don't line up with the actual detents so you've got a thrust change on A/THR DISC

Fixed in 3D branch, you can slew detents to where you need. Will make a nice gui for it soon

Kind Regards,
Josh

**Comment by axelkar** [comment]
Wouldn't this be good to have in FG itself?

**Comment by Octal450** [comment]
Not really doable.
Reverser stuff is custom.
